### PR TITLE
Documentation: Add documentation for `Nilai.php` #3

### PR DESCRIPTION
**Pull request by illusi0nary**
Added phpdoc + return types to the Nilai.php

It is not clear what autorefetch() returns, so i declared it as mixed.

### CODE REVIEW

**Comment by chez14** [comment]
> It is not clear what autorefetch() returns, so i declared it as mixed.

Good catch! This was my part of the error. I made some incosistencies back then. Will fix this in the future.

---

Just like other MR, we need more context to the functions. Could you please add more info on the docs?

- Nilai = Marks (in context of points)

File: src/Data/Nilai.php [hunk 1]
```diff
@@ -79,6 +86,11 @@ public function getSemesters($refetch = false)
         return Enums\Semester::createFromPreg($semester_data);
     }
 
+    /**
```
Comment (chez14) [inline src/Data/Nilai.php:92]: The function will automatically detect if something has been fetched before. Something like auto caching.

**Comment by illusi0nary** [comment]
I updated the PR regarding to your feedback.

**Comment by chez14** [comment]
Yay! Thanks!

Adding the label as per https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Pull request merged**